src/server/controllers/user.js

```diff
@@ -5,6 +5,7 @@ const prisma = new PrismaClient();
 
 const key = '87764d1a-92dc-4ced-a758-9c898c31d525'
 const saltRounds = 10;
+const maxAge = 7 * 24 * 60 * 60
```

Comment:
So this is the number of seconds in a week? Is this just because the data needs to be in seconds?

Reply:
exactly, for the JWT library, but if u wanna do it with cookies usually U will multiply it by 1000